tests/test_views.py

```diff
@@ -1,15 +1,22 @@
 import pytest
+from urllib.parse import urlencode
+import usaddress
 
 
 def test_api_parse_succeds(client):
-    # TODO: Finish this test. Send a request to the API and confirm that the
-    # data comes back in the appropriate format.
     address_string = '123 main st chicago il'
-    pytest.fail()
+    test_address_components, test_address_type = usaddress.tag(address_string)
+    get_param = urlencode({'address': address_string}) 
 
+    response = client.get(f'/api/parse/?{get_param}')
```

Comment:
You can submit URL parameters as an argument to `client.get()`, rather than encoding them yourself. See [the Django docs on testing tools](https://docs.djangoproject.com/en/dev/topics/testing/tools/#the-test-client) for an example.

Reply:
Thank you for that resource! I've updated the client get request to pass in the address string variable as the get query parameter.